### ISSUE TITLE
Browser Testing & Accessibility Testing 

### ISSUE THREAD

**Issue by pamxla** [TriNguyen479/web-design-2-independent-project#9]
Hi! 
I was looking at your website and it looks amazing!! Everything is well put together, and it works well with all screen sizes. The nav links and buttons all function well. I can open it in Safari and Firefox, but the only thing that doesn't show up is your favicon.

**Accessibility** 
While checking your accessibility, everything was good, you added skip links and background color behind the white text. I ran your website through the chrome accessibility test and the results were high, so I'm pretty sure you passed!. The speed of your website was also good and the voiceover works for your website. 
Great work! :D
![Screen Shot 2021-12-10 at 12 43 37 PM](https://user-images.githubusercontent.com/77292983/145619352-f04ee37c-4a02-49c2-b32c-5ee5fe389e01.png)

**Comment by pamxla** [comment]
Forgot to mention
I ran your website on WAVE https://wave.webaim.org/report#/https://tringuyen479.github.io/web-design-2-independent-project/ and says you just missing some alt text for your footer social icons
![Screen Shot 2021-12-10 at 1 02 57 PM](https://user-images.githubusercontent.com/77292983/145620938-ea171f3f-214b-450c-81fe-c25877a6d698.png)
 You should also check here https://www.webpagetest.org/performance_optimization.php?test=211210_AiDcP6_2970ab072eed8ec8775cf6c61da1f82d&run=1#cache_static_content
